Testing at Travis-CI is now available

greatfireball opened the pull request:
This addresses #7 

Due to the test run under Linux and fail under MacOSX we need to merge it to the newly created branch `feature/new-test-script`

<!---
@huboard:{"order":8.0,"milestone_order":8.0,"custom_state":""}
-->

greatfireball committed:
Create .travis.yml

Travis-CI test should run from now on

greatfireball committed:
Added testing under osx and linux to travis.yml

I am using the new feature to test under different operation systems.

Therefore, we should be able to guarantee that the SeqChunker works under Linux and MacOSX

greatfireball committed:
Skipping installation step

According to https://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Installation-Step the installation step to resolve dependencies can be avoided. In our project that makes sense due to we have no real perl program running

review-ninja commented:
<a href="https://app.review.ninja/BioInf-Wuerzburg/SeqChunker/pull/8" target="_blank"><img src="https://app.review.ninja/25783014/pull/8/badge" alt="ReviewNinja"/>

greatfireball committed:
Ignoring build failures under osx

Due to we are sure about the MacOSX version, I tollerate build failures under osx

greatfireball committed:
Changed to prove harnest to run tests

This is expected to fail for the original test script, but should work on the new one
